src/air/tests/test_autogenerated.cairo

```diff
@@ -76,166 +68,162 @@ fn test_eval_composition_polynomial_inner() {
 #[available_gas(9999999999)]
 fn test_eval_oods_polynomial_inner() {
     let column_values = array![
-        0x465cc38930fc5209ab9c13e30b61f166da9e900abfc26ac7ce2f840b121375a,
-        0x574340d33431f3096eb4d2f2a7abf22d95e36ec938b6627269fef207b618241,
-        0x7f1a146e05c096608ca7a7222a558776ba1e54551c23b13cd5b99477098b24a,
-        0x2be63af1ebc4b5e711c5531eb3b3f5a52d26c5fc825a3460bcf39c8c3def44c,
-        0xc1460285464e30adca071c213ae51fa7c06c3b53a5d8137ac9cd657c1af0d8,
-        0x5b9f7c683cee3855ae9aa71e3d6177e0909f67fbcd45721f861f7c7780142f5,
-        0x4208c4609b8535aa6d8016c971d0baaa41ae81fd757d52be5372aec063271c5,
-        0x79cbd5d9008c17acc9db4a02c8a61c348b3321d8c5fcaa977052f721ca6331e,
-        0x1ebf13deb07846470690aba50e67899c1b0d7879b0b4f80ae20e3ae8aac99c9,
-        0x78c7aab3b9642278a45a46a5b3d8c084e15b863db7fc88b8b446831136325b0,
-        0x29f2c6eb5bd093c9b6260c1070d21c6f96568681ecbd2a8e593fe88811c5cb3,
-        0x1003d9936322ad4c6218fe18d906b9e6b70c2146685c27fbf7488644a65c0bd,
+        0x5900b3cc21a91beadc7d227c7ab4de9d8666bbe9cef8cd3b53c96227636baa7,
+        0x364ec5a537535bfed197fa6d37ddc5059b46f7ddc3106e32c869c4af92801a4,
+        0x3e0ee23636524e70df1d826df10120ba276b4045fa3cc21f68b80aca8a6bcf6,
+        0x1615927ad08e7071d224fd62277c733856264b019f19348cc06d12c818347a1,
+        0x139bc4a27e309456740143e4d681d0482789ec7e6464b4fe0d7ed276d87c241,
+        0x6977d98591d84c22adecac4b4dc42a0b2e609c0daf993e538c9720521dbf02d,
+        0xfa852d9525f19f876b72d275a75b63beb87c7f5934285378e8163804ad58aa,
+        0x12c213443cac1d7138a5ecc07a4eca17719894b5503fe5f658186066951ec0b,
+        0x6bf897a3f79da8666994fad140c1d7916c7e27c9836856ca160c1a49b28b408,
+        0x5a4730e9f179e6bb22412d9a14eb5bfd8c8e5197b0fc0a962a94eb2d8542ea3,
+        0x2187d700f2a286da191941824f267f4199e07cc1707b5b346868dcb718c30e5,
+        0x497821407826448c240a13a2b2c2c15c47fbe074d29c2446a2bf63020483765
     ];
-    assert(column_values.len() == NUM_COLUMNS_FIRST + NUM_COLUMNS_SECOND + 2, 'Invalid value');
```

Comment:
What's the reason this has been removed?

Reply:
I decided to remove this and similar checks (for oods_values too)
It is fixed defined test, i think no point in checking those kind of things if u mess up values test is going to fail if u give to much it wont matter for the tested function either way